cptv-reprocess.py

```diff
@@ -61,10 +70,25 @@ def parse_args():
     parser.add_argument(
         "-id",
         dest="recording_id",
+        type=int,
         default=None,
         help="Specify the recording id to download",
     )
+    parser.add_argument(
+        "-upperid",
+        dest="upper_id",
+        type=int,
+        default=None,
+        help="Specify the recording upper id e.g Recording.id <= upper_id",
+    )
 
+    parser.add_argument(
+        "-lowerid",
+        dest="lower_id",
+        type=int,
+        default=None,
+        help="Specify the recording lower id value e.g Recording.id >= lower_id",
+    )
```

Comment:
Instead of having separate args for a single id and ranges, how about just having the `--id` arg take either a single id or a range in the format `start:end` or even `start:`. This can done by providiong a simple parsing function as `add_argument`'s `type` parameter. This would take the raw string value and return a (start, end) tuple. For a single recording the start and end would be the same.